registration-processor/pre-processor/registration-processor-operator-validator-stage/src/main/java/io/mosip/registration/processor/stages/operatorvalidator/OperatorValidator.java

```diff
@@ -361,16 +361,19 @@ private boolean validateMapping(List<String> pathsegments, InternalRegistrationS
 				RegistrationCenterUserMachineMappingHistoryResponseDto.class);
 		regProcLogger.debug("validateMapping call ended for registrationId {} with response data {}",
 				registrationStatusDto.getRegistrationId(), JsonUtil.objectMapperObjectToJson(userDto));
-		if (userDto != null) {
-			if (responseWrapper.getErrors() == null) {
-				isValidUser = userDto.getRegistrationCenters().get(0).getIsActive();
-			} else {
-				List<ErrorDTO> error = responseWrapper.getErrors();
-				regProcLogger.debug("validateMapping call ended for registrationId {} with error data {}",
-						registrationStatusDto.getRegistrationId(), error.get(0).getMessage());
-				throw new BaseCheckedException(error.get(0).getMessage(),
-						StatusUtil.CENTER_DEVICE_MAPPING_NOT_FOUND.getCode());
-			}
+		
+		if (responseWrapper.getErrors() != null) {
+			List<ErrorDTO> error = responseWrapper.getErrors();
+			regProcLogger.debug("validateMapping call ended for registrationId {} with error data {}",
+					registrationStatusDto.getRegistrationId(), error.get(0).getMessage());
+			throw new BaseCheckedException(error.get(0).getMessage(),
+					StatusUtil.CENTER_DEVICE_MAPPING_NOT_FOUND.getCode());
+		} else if (userDto != null) {
+			isValidUser = userDto.getRegistrationCenters().get(0).getIsActive();
+		} else {
+			regProcLogger
+					.debug("validateMapping call ended for registrationId with error not there and userDTO is empty");
```

Comment:
Make this as warning log and change the message to say that "validateMapping call ended with no erros and userDTO as null so considering as mapping not valid"

Please take care of the line formatting

Reply:
changed the log message.